projects/extension/tests/text_to_sql/test_dump_restore.py

```diff
@@ -20,6 +21,21 @@ def db_url(user: str, dbname: str) -> str:
     return f"postgres://{user}@127.0.0.1:5432/{dbname}"
 
 
+# skip tests in this module if this is NOT a pre-release build
+with psycopg.connect(
+    db_url(user="postgres", dbname="postgres"), autocommit=True
+) as con:
+    with con.cursor() as cur:
+        cur.execute(
+            """select default_version from pg_available_extensions where name = 'ai'"""
+        )
+        version = cur.fetchone()[0]
+        parts = re.split(r"[.-]", version, maxsplit=4)
+        if len(parts) == 3:  # if this is a release version
```

Comment:
```suggestion
        if len(parts) == 3:  # if this is not a release version
```

Reply:
actually, it's correct as-is. There will be 3 parts to the version if it is a release version `1.2.3`. There are 4 parts to a pre-release version `1.2.3-4`. We want to skip these tests if it is a release version. We want to run them if it is a pre-release version